config/initializers/pagy.rb

```diff
@@ -145,8 +145,8 @@
 
 # Overflow extra: Allow for easy handling of overflowing pages
 # See https://ddnexus.github.io/pagy/docs/extras/overflow
-# require 'pagy/extras/overflow'
-# Pagy::DEFAULT[:overflow] = :empty_page    # default  (other options: :last_page and :exception)
+require 'pagy/extras/overflow'
```

Comment:
**question:** Just curious on why you had to uncomment this, I could not see it being used.